CHANGELOG.rst

```diff
@@ -6,6 +6,11 @@ fourfront
 Change Log
 ----------
 
+4.5.18
+======
+
+* Added a new user insert in master_inserts/user.json
```

Comment:
Can you add your first name here so we can more easily track i.e. '... user insert for Rahi ...'